test/spec/modules/edgequeryxBidAdapter_spec.js

```diff
@@ -0,0 +1,116 @@
+import {
```

Comment:
any specific reason why you use this multi-line format for imports here, while on lines 13 and 14 you're inlining them?